pkg/distributor/distributor_test.go

```diff
@@ -65,7 +65,7 @@ import (
 )
 
 var (
-	errFail                    = httpgrpc.Errorf(http.StatusInternalServerError, "Fail")
+	errFail                    = status.Error(codes.Internal, "Fail")
```

Comment:
Why this change?

Reply:
`errFail` is used in `mockIngester` struct as an example of error returned by that ingester client. After the recent error handling refactoring, it is expected that ingesters will not return errors with HTTP status codes anymore. This is still the default behaviour, but will soon be deprecated.
As a consequence, I replaced the current `errFail` value with an error with a pure gRPC code. 
It is worth noting that this change could be done in a separate PR.